src/temp/ProjectViewTabs/ProjectViewTabs.module.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../theme/theme';
+.tabsContainer {
+display: flex;
+width: fit-content;
+height: 48px;
+border-radius: 12px;
+background: $backgroundColor;
+align-items: center;
+justify-content: center;
+padding: 0px 5px;
+gap: 16px;
+position: fixed;
+left: 10px;
+right: auto;
+margin: 0 auto;
+top: 100px;;
+z-index: 10;
+}
+.option { 
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    font-weight: 700;
+    cursor: pointer;
+    width: 50px;
+    height: 40px;
+    border-radius: 8px;
+    flex: 1;
+    gap:6px;
+  }
```

Comment:
Should the button size be fixed, or the padding? The figma design fixes the padding.

Reply:
Yes, the implementation is consistent with the old implementation where we've fixed the padding and size. Also, the button's size and padding would remain same for all the sizes.